client_src/js/fileManagerRunner.js

```diff
@@ -174,8 +173,13 @@ class Manager {
 
   static changePrinter(e, target) {
     if (!e.target.id.includes("filamentDrop")) {
+      const filamentList = document.getElementById("fileBody");
+      if (!!filamentList) {
+        filamentList.innerHTML = "";
+        filamentList.scrollTop = 0;
+      }
+
```

Comment:
I can gather this is my earlier goof but filamentList?

Reply:
What was I smoking XD

Reply:
Oooh this is all me, except for the line before where "filamentDrop" is shown. You know those situations where contextual words make you type the wrong thing? This is my goof hahaha. Will fix later, as well as your other comments.

Reply:
Omg, not me! ahaha no worries